File: src/ERP/acme.atena.domain/DTO/Account/FluxoDeCaixaReceita.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace acme.atena.domain.DTO.Account
{
    public class FluxoDeCaixaReceita : AbstractEntity
    {
        public Guid  FluxoDeCaixaId { get; set; }
        public Guid ReceitaId { get; set; }

        public virtual FluxoDeCaixa FluxoDeCaixa { get; set; }
        
        public virtual Receita Receita { get; set; }
    }
}

File: site/coreui-free-angular-admin-template-master/src/app/view-model/response/account/fluxo-de-caixa-receita.ts
import { Receita } from './receita';
import { AbstractEntity } from "../abstract-entity";
import { FluxoDeCaixa } from "./fluxo-de-caixa";

export class FluxoDeCaixaReceita extends AbstractEntity{
  public  fluxoDeCaixaId :string;
  public  receitaId :string;

  public fluxoDeCaixa: FluxoDeCaixa;
  public receita :Receita;
}
